src/test/java/software/amazon/awssdk/crt/test/Mqtt5ClientTest.java

```diff
@@ -120,6 +123,10 @@ private void populateTestingEnvironmentVariables() {
         if (System.getenv("AWS_TEST_MQTT5_PROXY_PORT") != null) {
             mqtt5ProxyPort = Long.parseLong(System.getenv("AWS_TEST_MQTT5_PROXY_PORT"));
         }
+        mqtt5ProxyMqttHost = System.getenv("AWS_TEST_MQTT5_PROXY_MQTT_HOST");
+        if (System.getenv("AWS_TEST_MQTT5_PROXY_PORT") != null) {
```

Comment:
Would this be "AWS_TEST_MQTT5_PROXY_MQTT_PORT"?

Reply:
Good catch! Will fix 👍